.github/dependabot.yml

```diff
@@ -5,108 +5,95 @@
 
 version: 2
 updates:
-  # Maintain dependencies for GitHub Actions Workflows
+
+  # Docker
+  - package-ecosystem: "docker"
+    directory: "/"
```

Comment:
I do not believe dependabot will [search subdirectories/subfolders](https://github.com/dependabot/dependabot-core/issues/1015).  Each location with docker files needs to be called out individually.